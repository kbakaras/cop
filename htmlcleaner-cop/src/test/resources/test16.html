<html><head></head><body>
  
    <div onclick='click("animation")'>
    </div>
  
</body></html>