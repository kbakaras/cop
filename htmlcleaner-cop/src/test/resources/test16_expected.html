<html><head></head><body>
  
    <div onclick="click(&quot;animation&quot;)">
    </div>
  
</body></html>